kolibri/plugins/coach/assets/src/routes/planExamRoutes.js

```diff
@@ -49,6 +49,23 @@ export default [
       {
         name: PageNames.QUIZ_SELECT_RESOURCES,
         path: ':section_id/select-resources',
+
+        children: [
```

Comment:
As a general rule, nested routes are a bad idea - I've tried them, and yet to find a particularly good reason for using them.

The main place they might have value is when you need to swap out different components within another view, as per the example in the docs: https://v3.router.vuejs.org/guide/essentials/nested-routes.html

However, that's not what you're doing here, where it's just being used to gate between the base route (show channels) and a specific topic route (show a topic). This can be better achieved by just using an optional URL parameter for topic_id in the route above. @AllanOXDi has implemented this already, I believe.